packages/app/features/player/components/likesLIst.tsx

```diff
@@ -0,0 +1,36 @@
+import { User } from "app/api/graphql";
+import { Text, View } from "app/design-system";
+import { FavoriteButton } from "app/ui/beat-list-entry/favoriteButton";
+import { UserAvatar } from "app/ui/userAvatar";
+import { FlatList } from "react-native";
+
+type Props = {
+  likers: User[];
+};
+
+export function LikesList({ likers }: Props) {
+  const renderItem = ({ item }: { item: User }) => {
+    return (
+      <View className="mr-2" key={item.id}>
+        <UserAvatar user={item} />
+      </View>
+    );
+  };
```

Comment:
(Optional) I would extract it outsize LikesList component

Reply:
As the `LikesList` component is quite small, I think we can leave it there.